apps/src/blockly/eventHandlers.ts

```diff
@@ -75,10 +88,11 @@ function updateBlockEnabled(block) {
         child.setEnabled(true);
       }
     } else if (block.outputConnection || block.previousConnection) {
+      let currentBlock: Block | null = block;
```

Comment:
the passed in parameter has to be a `Block`, but we want to support a variable here that could also be `null`